src/core_plugins/metrics/public/components/vis_types/timeseries/vis.js

```diff
@@ -71,14 +76,28 @@ function TimeseriesVisualization(props) {
           const rowSum = seriesData.reduce((acc, item) => {
             return item.data[index][1] + acc;
           }, 0);
-          seriesData.forEach(item => {
-            item.data[index][1] = rowSum && item.data[index][1] / rowSum || 0;
-          });
+          percentSums[index] = percentSums[index] && percentSums[index] + rowSum || rowSum;
```

Comment:
before you were checking if rowSum is defined ... what changed so you can now assume its defined ?
also writing this with if would make it more readable imo